packages/react-server/src/server/server.ts

```diff
@@ -17,10 +17,12 @@ interface Options {
   ip?: string;
   port?: number;
   assetPrefix?: string;
+  proxy?: boolean;
   assetName?: string | ValueFromContext<string>;
   serverMiddleware?: compose.Middleware<Context>[];
   render: RenderFunction;
   renderError?: RenderFunction;
+  app?: Koa;
```

Comment:
It feels a little bit like we are only doing this for testing. But I also dont see anything too wrong with it. So 🙆‍♀️